rust/fuzz/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+
+[package]
+name = "suricata-fuzz"
+version = "0.0.1"
+authors = ["Automatically generated"]
+publish = false
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies.suricata]
+path = ".."
+[dependencies.libfuzzer-sys]
+git = "https://github.com/rust-fuzz/libfuzzer-sys.git"
+
```

Comment:
Is this the answer to my question here 
https://github.com/google/oss-fuzz/issues/2145#issuecomment-485781098